Lib/enum.py

```diff
@@ -309,6 +309,10 @@ def __call__(cls, value, names=None, *, module=None, qualname=None, type=None, s
         return cls._create_(value, names, module=module, qualname=qualname, type=type, start=start)
 
     def __contains__(cls, member):
+        if not isinstance(member, Enum):
+            raise TypeError(
+                "Unsupported operands type(s) for 'in': '%s' and '%s'" % (
+                    type(member).__qualname__, type(Enum).__qualname__))
```

Comment:
I just noticed that if we use `cls.__qualname__` instead of `type(Enum).__qualname__`, we get a much better output.

For example if we have the following program
```py
ABC = enum.Flag('ABC', 'a, b, c')
'test' in ABC
```
then after changing to `cls.__qualname__`, we get:
```
TypeError: Unsupported operands type(s) for 'in': 'str' and 'ABC'
```
which looks better than what we were getting before:
```
TypeError: Unsupported operands type(s) for 'in': 'str' and 'EnumMeta'
```

Which one should we use?

Reply:
That definitely looks better; go ahead and use `cls.__qualname__` where ever it's needed.